src/pull-tutorial.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+GIT_BRANCH="master"
+TUTORIAL_NAME=${1}
+USER="jovyan"
+
+if [ -z "${TUTORIAL_NAME}" ] ; then
+    echo Usage: ${0} \<turtorial name\>
+    exit 1
+fi
+
+function main() {
+  create_working_dir
+  download_tutorial_docs
+  move_data_to_user_work_dir
+  prep_user
+  cleanup
+  exit 0
+}
+
+function create_working_dir() {
+  mkdir -p /tmp/.cache
+  cd /tmp/.cache
+}
+
+function download_tutorial_docs() {
+  FILE_LIST=$( curl -so - https://raw.githubusercontent.com/statgenetics/statgen-courses/${GIT_BRANCH}/handout/_${TUTORIAL_NAME}.txt )
+  for URL in ${FILE_LIST} ; do
+    FILE=${URL##*/}
+    curl -so ${FILE} ${URL}
+    if [[ ${FILE} == *.ipynb ]] ; then
```

Comment:
do you want to then remove the nbconvert in the 3 docker files vat pseq and finemap?